### PR TITLE
add page for pxctl auth

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px pls review when you have some time. Also, lmk if there are any other subcommands.

### CODE REVIEW

**Comment by andreipope** [comment]
@harsh-px please review when you have some time

File: content/reference/CLI/authorization.md [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
 ## Overview
 
-This document outlines how to interact with an auth-enabled _PX_ cluster. The main way to do it is by using the `pxctl context` commands. In addition, you can integrate with an __OIDC provided token__ or __generate self-signed tokens__ through `pxctl`.
+This document outlines how to interact with an authorization-enabled Portworx cluster. The main way to do it is by using the `pxctl context` command. Also, you can integrate with an __OIDC provided token__ or __generate self-signed tokens__ through `pxctl`. See the [generate self-signed tokens](/reference/cli/self-signed-tokens) page for more details.
 
-## Context
+## Contexts
 
-`pxctl` allows you to store contexts and associated clusters, privileges, and tokens local to your home directory. This way, you can easily switch between these configurations with a few commands.
+Portworx stores your:
```
Comment (nathan-px) [inline content/reference/CLI/authorization.md]: We shouldn't break a sentence in the middle with a bulleted list. format it like this instead: 

```suggestion
Portworx stores the following locally to your home directory, allowing you to switch between configurations with a few commands:

 - contexts
 - associated clusters
 - privileges
 - tokens
```

File: content/reference/CLI/authorization.md [hunk 2]
```diff
@@ -77,76 +90,57 @@ contextconfig:
     endpoint: http://localhost:9001
 ```
 
+### The current context
 
-### Current context
-
-Now that you've created your contexts, you can easily switch between them with the commands below. `pxctl` will automatically read your currently set context and use the associated token for all commands.
+Once you created your contexts, use the `pxctl context` command to switch between them. Under the hood, Portworx reads your current context and then uses the associated token for all commands.
```
Comment (nathan-px) [inline content/reference/CLI/authorization.md]:
```suggestion
Once you've created your contexts, use the `pxctl context` command to switch between them. Under the hood, Portworx reads your current context and then uses the associated token for all commands.
```

File: content/reference/CLI/authorization.md [hunk 3]
```diff
@@ -77,76 +90,57 @@ contextconfig:
     endpoint: http://localhost:9001
 ```
 
+### The current context
```
Comment (nathan-px) [inline content/reference/CLI/authorization.md]:
```suggestion
### Set the current context
```

File: content/reference/CLI/authorization.md [hunk 4]
```diff
@@ -77,76 +90,57 @@ contextconfig:
...
 
-* [Keycloak](https://www.keycloak.org/docs/latest/server_development/index.html#admin-rest-api)
-* [Auth0](https://auth0.com/docs/api/authentication#get-token)
-* [Okta](https://developer.okta.com/docs/api/getting_started/getting_a_token/#token-expiration)
+- OIDC and
+- self-signed.
 
-For self-signed, you can use your own JWT compliant application, or for
-convenience, `pxctl` has a command for generating tokens.
+To generate a token through your OIDC provider, see the documentation on generating bearer tokens on the website of the provider. As a reference, here is the relevant documentation for the most commonly used OIDC providers:
```
Comment (nathan-px) [inline content/reference/CLI/authorization.md]:
```suggestion
To generate a token through your OIDC provider, see the documentation on generating bearer tokens on the website of the provider. The following links direct you to the most commonly used OIDC providers:
```

File: content/reference/CLI/self-signed-tokens.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      --context string   context name that overrides the current auth context
+  -j, --json             output in json
+      --key string       path to client key for ssl usage
+      --raw              raw CLI output for instrumentation
+      --ssl              ssl enabled for portworx
+```
+
+As seen above, the `pxctl` command-line tool allows you to generate self-signed tokens in the following ways:
+
+- ECDSA,
```
Comment (nathan-px) [inline content/reference/CLI/self-signed-tokens.md]:
```suggestion
- ECDSA
```

File: content/reference/CLI/self-signed-tokens.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+- RSA,
```
Comment (nathan-px) [inline content/reference/CLI/self-signed-tokens.md]:
```suggestion
- RSA
```

**Pull request merged**